lib/tasks/funding.rake

```diff
@@ -28,4 +28,9 @@ namespace :funding do
     missing_urns = Funding::LeadPartnerTraineeSummariesImporter.call(attributes:)
     abort("Lead school URNs: #{missing_urns.join(', ')} not found") unless missing_urns.empty?
   end
+
+  desc "Set the route_type to the corresponding translation key defaults to academic_year 2024/2025 unless specified: rails funding:generate_trainee_summary_row_route_type or 'funding:generate_trainee_summary_row_route_type[2024/2025]'"
```

Comment:
```suggestion
  desc "Set the route_type to the corresponding translation key defaults to academic_year 2024/2025 unless specified: rails funding:generate_trainee_summary_row_route_type or 'funding:generate_trainee_summary_row_route_type[2024/25]'"
```

Sense check was it meant to be 
`2024/25` vs 2024/2025

Reply:
Good spot